staging/src/k8s.io/apiserver/pkg/server/config.go

```diff
@@ -340,6 +344,9 @@ func NewConfig(codecs serializer.CodecFactory) *Config {
 		LongRunningFunc:       genericfilters.BasicLongRunningRequestCheck(sets.NewString("watch"), sets.NewString()),
 		APIServerID:           id,
 		StorageVersionManager: storageversion.NewDefaultManager(),
+
+		// use the default for now.
```

Comment:
Remove the comment - the "Default" in the name already clearly explains what we do :)